sklearn/utils/tests/test_validation.py

```diff
@@ -223,6 +223,9 @@ def test_check_array_links_to_imputer_doc_only_for_X(input_name, retype):
         " data, for instance by using an imputer transformer in a pipeline"
         " or drop samples with missing values. See"
         " https://scikit-learn.org/stable/modules/impute.html"
+        " You can find a list of all estimators that handle NaN values"
+        " by following this link to the documentation "
+        " above and going to section 6.4.7"
```

Comment:
You can also directly link the paragraph as
https://scikit-learn.org/stable/modules/impute.html#estimators-that-handle-nan-values

Reply:
Indeed, the numbering in the section would be brittle.